looty/src/main/scala/looty/poeapi/PoeRpcs.scala

```diff
@@ -243,7 +243,10 @@ object PoeRpcs {
 
   private var willCheckQueue = false
   //How long to wait after we hit the throttle before checking again
-  val coolOffMs = 10000
+  val coolOffMs = 60000
```

Comment:
ok so the reason i used 10 seconds (10000 ms) was that if i used 60 seconds, and it was very slightly early, and got rejected by GGG then i would wait 60 more seconds before trying again which is not great. by polling every 10 seconds the max wait is probably 70 seconds (as opposed to 120 seconds).